source/OpenCTM/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+
+add_subdirectory(${MESHLIB_THIRDPARTY_DIR}/OpenCTM-git ./OpenCTM)
```

Comment:
Let us delete source/OpenCTM folder completely and take everything from submodule